pom.xml

```diff
@@ -343,16 +343,13 @@
         <legacy.version.com.squareup.okhttp>3.14.9</legacy.version.com.squareup.okhttp>
```

Comment:
@yersan All the version props are in the root pom, e.g. https://github.com/wildfly/wildfly/blob/main/boms/standard-test/pom.xml has none.

We could move them out, to follow what we did in core's testbom. A subtle difference is here we have > 1 such test bom, so moving them out means dispersing them a bit more than is the case in core. OTOH it is clearer what a prop is used for if it's not in the root, and it keeps core and full consistent in how it's handled.

I'm fine with doing moving them. I'd prefer to do it separately though, as handling boms/standard-test is out of scope for this JIRA. Plus I want to get it merged and not wait for any discussion of the move.

Reply:
@bstansberry I did not explain my concern properly. I'm fine with the place where the properties are being declared, my concern is about the place where the properties are used to declare the dependency. In any case, It was unclear in my yesterday's comment.

`legacy.version.com.squareup.okhttp` property version is used directly in [testsuite/integration/clustering/pom.xml](https://github.com/wildfly/wildfly/blob/0c4d0b3a30c6090505391cec8bbf509b278eb638/testsuite/integration/clustering/pom.xml#L179C6-L184)

Since the boms/legacy-test/pom.xml description is the following:

"Bill of materials for testsuites that use legacy version of artifacts that differ from the versions used in the current release."

I was just wondering the reason why we don't declare `com.squareup.okhttp` dependency in the boms/legacy-test/pom.xml and import it in testsuite/integration/clustering/pom.xml by using this test bom. The same is applicable to `org.jboss.narayana` which is declared in testsuite/integration/legacy-ejb-client/pom.xml.

I guess you don't want to pollute the classpath of testsuite/integration/clustering and testsuite/integration/legacy-ejb-client with other dependencies that could be incorporated from the test-bom, so this is more a question to understand the reason more than a request to change anything.

Reply:
@yersan Thanks for the thorough review!

Re com.squareup.okhttp, it indeed is because I didn't want to pollute the testsuite/integration/clustering/pom.xml classpath. Maybe it would work but I didn't want to spend the time to try. :). I suspect it would introduce problems though.

Re: narayana-jts-idlj, it turns out I could have put that one in legacy-test and then imported that bom into testsuite/integration/legacy-ejb-client. That's a fast TS to run so I tried it and it passes and I was about to add a commit and push, but then I remembered https://github.com/wildfly/wildfly/pull/16481, which removes that entire testsuite! So I'll just leave that alone, and I'll make sure #16481 deletes the version property.